ui/app/components/ui/item/item.component.js

```diff
@@ -0,0 +1,141 @@
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import Icon from '../icon'
+
+const SendIcon = () => (
+  <Icon
+    type="send"
+    width={28}
+    height={28}
+    color="#2F80ED"
+  />
+)
+
+const InteractionIcon = () => (
+  <Icon
+    type="interaction"
+    width={28}
+    height={28}
+    color="#2F80ED"
+  />
+)
+
+const ApproveIcon = () => (
+  <Icon
+    type="approve"
+    width={28}
+    height={28}
+    color="#2F80ED"
+  />
+)
+
+const FailIcon = () => (
+  <Icon
+    type="interaction"
+    width={28}
+    height={28}
+    color="#D73A49"
+  />
+)
+
+const Item = ({
+  className,
+  status,
+  title,
+  subtitle,
+  more,
+  crypto,
+  cash,
+}) => {
+  const isApproved = status === 'approved'
+  const isPending = status === 'pending'
+  const isFailed = status === 'failed'
+
+  if (status === 'unapproved') {
+    subtitle = (
+      <h3>
+        <span className="unapproved">Unapproved</span> · {subtitle}
+      </h3>
+    )
+  }
+
+  if (isFailed) {
+    subtitle = (
+      <h3>
+        <span className="failed">Failed</span> · {subtitle}
+      </h3>
+    )
+  }
+
+  return (
+    <div className={className}>
+      <div className="col icon">
+        {isApproved ? (
+          <ApproveIcon />
+        ) : isPending ? (
+          <InteractionIcon />
+        ) : isFailed ? (
+          <FailIcon />
+        ) : (
+          <SendIcon />
+        )}
+      </div>
+      <div className={`col main${isApproved ? ' approved' : ''}`}>
+        {typeof title === 'string' ? (
+          <h2>
+            { title } {isPending && (
+              <span>
+                <Icon
+                  className="preloader"
+                  type="preloader"
+                  width={16}
+                  height={16}
+                  color="#D73A49"
+                />
+              </span>
+            )}
+          </h2>
+        ) : (
+          title
+        )}
+        {typeof subtitle === 'string' ? (
+          <h3>{ subtitle }</h3>
+        ) : (
+          subtitle
+        )}
+        {more && (
+          <div className="more">
+            { more }
+          </div>
+        )}
+      </div>
+      <div className={`col amount${isApproved ? ' approved' : ''}`}>
+        <h2>{crypto}</h2>
+        <h3>{cash}</h3>
+      </div>
+    </div>
+  )
+}
+
+Item.defaultProps = {
+  status: 'pending',
+}
+
+Item.propTypes = {
+  className: PropTypes.string,
+  status: PropTypes.string,
+  title: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.node,
+  ]).isRequired,
+  subtitle: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.node,
+  ]).isRequired,
+  more: PropTypes.node,
+  crypto: PropTypes.string,
```

Comment:
The prop names "crypto" and "cash" are not quite reflective of what is possible in the extension. Users can set their desired currency conversions on the settings screen, and then select whether they want eth or the selected one to be primary. This makes it possible for both currencies to be cryptos.

See screenshots

![Screenshot from 2020-01-15 11-51-58](https://user-images.githubusercontent.com/7499938/72446353-fd6e4480-378d-11ea-9e08-51c38c099bbc.png)
![Screenshot from 2020-01-15 11-50-09](https://user-images.githubusercontent.com/7499938/72446354-fd6e4480-378d-11ea-81f5-8f224d83fb7f.png)

In other places in we've used "nativeCurrency" for ETH and "currentCurrency" for whatever the user has selected.